lib/ansible/config/module_defaults.yml

```diff
@@ -1237,8 +1237,6 @@ groupings:
   - vmware
   vmware_deploy_ovf:
   - vmware
-  vmware_dns_config:
```

Comment:
You should keep the old alias here until it's entirely removed. This will break existing playbooks until switching to the new name is mandatory.

Reply:
ah good catch, thanks, I misread the vmware_host_dns addition as being a re-ordering of this one - thanks @s-hertel.